### DISCUSSION TITLE
filament change temperature setting

### DISCUSSION THREAD

**Discussion by jxer** [mriscoc/Ender3V2S1#415] [General]
where is this setting store? it defaults to 180, is this adjustable via menu?

**Comment by mriscoc** [comment]
You can pre-heat to the desired temperature, if you don't set a temperature, the system will pre-heat to the extruding minimum temperature.
You can set this temperature in the Filament settings menu, but be aware that increasing that temperature will prevent extrusion if the hot end temperatura drops below that setting.

**Comment by jxer** [comment]
any chance to get this in the filament change menu as a stored setting? petg won't flow at 180, it seems like a lot of extra steps to get over to control when this is something that could just be set once.

**Reply by mriscoc** [reply]
It could be in a future version.
